test/spec/modules/relevatehealthBidAdapter_spec.js

```diff
@@ -0,0 +1,204 @@
+      }
+    };
+  });
+
+  describe('validations', function () {
+    it('isBidValid : placementId and user_id are passed', function () {
+      let bid = {
+          bidder: 'relevatehealth',
+          params: {
+            placementId: 110011,
+            user_id: '11211'
+          }
+        },
+        isValid = spec.isBidRequestValid(bid);
+      expect(isValid).to.equals(true);
+    });
+    it('isBidValid : placementId and user_id are not passed', function () {
+      let bid = {
+          bidder: 'relevatehealth',
+          params: {
+            width: 160,
+            height: 600,
+            domain: '',
+            bid_floor: 0.5,
+            user_id: ''
+          }
+        },
+        isValid = spec.isBidRequestValid(bid);
+      expect(isValid).to.equals(false);
+    });
+  });
+  describe('Validate Request', function () {
+    it('Immutable bid request validate', function () {
+      let _Request = utils.deepClone(request),
+        bidRequest = spec.buildRequests(request);
+      expect(request).to.deep.equal(_Request);
+    });
+    it('Validate bidder connection', function () {
+      let _Request = spec.buildRequests(request);
+      expect(_Request.url).to.equal('https://rtb.relevate.health/prebid/relevate');
+      expect(_Request.method).to.equal('POST');
+      expect(_Request.options.contentType).to.equal('application/json');
+    });
+    it('Validate bid request : Impression', function () {
+      let _Request = spec.buildRequests(request);
+      let data = JSON.parse(_Request.data);
+      //   expect(data.at).to.equal(1); // auction type
+      expect(data[0].imp[0].id).to.equal(request[0].bidId);
+      expect(data[0].placementId).to.equal(110011);
+    });
+    it('Validate bid request : ad size', function () {
+      let _Request = spec.buildRequests(request);
+      let data = JSON.parse(_Request.data);
+      expect(data[0].imp[0].banner).to.be.a('object');
+      expect(data[0].imp[0].banner.w).to.equal(160);
+      expect(data[0].imp[0].banner.h).to.equal(600);
+    });
+    it('Validate bid request : user object', function () {
+      let _Request = spec.buildRequests(request);
+      let data = JSON.parse(_Request.data);
+      expect(data[0].user).to.be.a('object');
+      expect(data[0].user.id).to.be.a('string');
+    });
+    it('Validate bid request : CCPA Check', function () {
+      let bidRequest = {
+        uspConsent: '1NYN'
+      };
+      let _Request = spec.buildRequests(request, bidRequest);
+      let data = JSON.parse(_Request.data);
+      expect(data[0].us_privacy).to.equal('1NYN');
+      //   let _bidRequest = {};
+      //   let _Request1 = spec.buildRequests(request, _bidRequest);
+      //   let data1 = JSON.parse(_Request1.data);
+      //   expect(data1.regs).to.equal(undefined);
+    });
+  });
+  describe('Validate response ', function () {
+    it('Validate bid response : valid bid response', function () {
+      let bRequest = spec.buildRequests(request);
```

Comment:
Correct me if I'm wrong, but it doesn't seem like setting bRequest and data is actually doing anything in this test. Remove these two lines.

Reply:
Yes, you are right. Removed the lines.